.npmignore

```diff
@@ -0,0 +1,14 @@
+node_modules/*
+es*/compilers/*
+.DS_Store
+.ruby-version
+_site/
+.idea/
+es5/
```

Comment:
Why wouldn't they need the HTML files if they wanted to display them, or if they want to be able to view the HTML pages without an internet connection?